### ISSUE TITLE
Add feature tag to alias

### ISSUE THREAD

**Issue by orhanerday** [mozilla/fx-private-relay#1384]
My request is; add tags to aliases to remember what they were created for.

![image](https://user-images.githubusercontent.com/22305274/143185200-c8f653d3-2d60-4bae-be32-aefdf2638d30.png)

**Comment by Vinnl** [comment]
Hi @orhanerday, this [is already possible](https://support.mozilla.org/en-US/kb/use-labels-keep-your-aliases-organized-firefox-rel), but requires you to [explicitly allow us to store those labels](https://support.mozilla.org/en-US/kb/firefox-relay-get-your-labels-all-your-devices). (Alternatively, you can install the browser extension, but then they'll only be available on the browser that has that extension installed.)

Feel free to re-open if you have further questions :)

**Comment by Gowee** [comment]
Hi, @Vinnl. 

The label is extremely useful. But I suppose few users could find that option "hidden" in the settings page unless they are searching for issues like my doing. It might be better if this label feature is an opt-out instead of an opt-in. Or, a prominent notification bar in the dashboard page would also be great to inform users. 

Thank you for providing this excellent Relay service.

**Comment by Oryoro** [comment]
If they are only stored in the browser plugin-in, would you be able to have a back-up of the labels by backing up your firefox profile folder? How about if the labes _are_ stored on the account, could you still back-up using the same method?

**Comment by Vinnl** [comment]
Hi @Gowee, het option actually is opt-out now; however, we didn't want to automatically opt-in people who were using Relay already before we introduced this feature, and were assuming that we did not store that data. (And we did display a notification bar in the dashboard to announce the new feature for a while.)

@Oryoro I'm not 100% sure, but yes, I think they are stored in your profile folder if you have server-side storage turned off, so backing up your profile folder should back up those labels too. If they _are_ stored on the server, then technically they're not in your profile folder, but if you restore a back-up of your profile folder then it will still fetch the same labels from the server, so in that sense they're backed up.
